kotlin/scene-layer/README.metadata.json

```diff
@@ -6,16 +6,25 @@
         "scene-layer.png"
     ],
     "keywords": [
-        "Layers",
+        "layer",
+	"scene",
+	"3D",
+        "ArcGISScene",
+	"ArcGISSceneLayer",
+	"ArcGISTileElevationSource",
+        "SceneView",
+        "Surface"	
```

Comment:
missing a comma after "Surface" here

Reply:
sorry, tabs instead of spaces for my edits...fixed!